Plugins/GeoModel/src/GeoModelDetectorObjectFactory.cpp

```diff
@@ -162,8 +162,18 @@ void Acts::GeoModelDetectorObjectFactory::convertFpv(
   std::vector<GeoModelSensitiveSurface> sensitives;
 
   for (const auto &surface : surfaces) {
-    const Transform3 &transform =
-        fpv->getAbsoluteTransform() * surface.transform;
+    const Transform3 &absTrans =
+        Eigen::Isometry3d(fpv->getAbsoluteTransform().matrix());
```

Comment:
This doesn't work:
```suggestion
    Transform3 absTrans{fpv->getAbsoluteTransform().matrix()};
```

Also, maybe you can remove the const refs? I think this is safe due to lifetime extension, but it could just be an lvalue honestly.

Reply:
I can remove the const ref, that's fine. 
But I think I still need to wrap fpv->getAbsoluteTransform().matrix() inside Eigen::Isometry3d. I can check.

Reply:
Your suggestion should work. I've changed as you suggested, see next comment.